PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -9,22 +11,34 @@ public static void Main(string[] args)
         {
             string filePath = GetFilePath();
             Question[] questions = LoadQuestions(filePath);
-
+            
+            Question[] questionsRnd = RandomizeQuestions(questions);
+            
             int numberCorrect = 0;
-            for (int i = 0; i < questions.Length; i++)
+            
+            // randomized ( questions updated to be questionsRnd )
+            for (int i = 0; i < questionsRnd.Length; i++)
             {
-                bool result = AskQuestion(questions[i]);
+                bool result = AskQuestion(questionsRnd[i]);
                 if (result)
                 {
                     numberCorrect++;
                 }
             }
-            Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
+            Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questionsRnd.Length) + " correct");
+            
         }
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            double ret = (((double)numberCorrectAnswers / (double)numberOfQuestions) * 100);
+            
+            if (ret == 0.0)
+            {
+                return "0%";
+            }
+            
+            return ret.ToString("#.##") + "%";
```

Comment:
You don't actually need to do any rounding, but a better way to do this would be as follows:

```suggestion
            return Math.Round(result, 2) + "%";
```